components/brave_wallet_ui/nft/components/nft-details/nft-details.tsx

```diff
@@ -0,0 +1,151 @@
+          </NftImageWrapper>
+          {!isImageLoaded &&
+            <LoadingSkeleton wrapper={NFTImageSkeletonWrapper} />
+          }
+          <DetailColumn>
+            <TokenName>
+              {selectedAsset.name} {
+                selectedAsset.tokenId
+                  ? '#' + new Amount(selectedAsset.tokenId).toNumber()
+                  : ''
+              }
+            </TokenName>
+            {/* TODO: Add floorFiatPrice & floorCryptoPrice when data is available from backend: https://github.com/brave/brave-browser/issues/22627 */}
+            {/* <TokenFiatValue>{CurrencySymbols[defaultCurrencies.fiat]}{nftMetadata.floorFiatPrice}</TokenFiatValue> */}
+            {/* <TokenCryptoValue>{nftMetadata.floorCryptoPrice} {selectedNetwork.symbol}</TokenCryptoValue> */}
+            <DetailSectionRow>
+              <DetailSectionColumn>
+                <DetailSectionTitle>{getLocale('braveWalletNFTDetailBlockchain')}</DetailSectionTitle>
+                <DetailSectionValue>{nftMetadata.chainName}</DetailSectionValue>
+              </DetailSectionColumn>
+              <DetailSectionColumn>
+                <DetailSectionTitle>{getLocale('braveWalletNFTDetailTokenStandard')}</DetailSectionTitle>
+                <DetailSectionValue>{nftMetadata.tokenType}</DetailSectionValue>
+              </DetailSectionColumn>
+              <DetailSectionColumn>
+                <DetailSectionTitle>{getLocale('braveWalletNFTDetailTokenID')}</DetailSectionTitle>
+                <ProjectDetailIDRow>
+                  <DetailSectionValue>
+                    {
+                      selectedAsset.tokenId
+                        ? '#' + new Amount(selectedAsset.tokenId).toNumber()
+                        : ''
+                    }
+                  </DetailSectionValue>
+                  <ExplorerButton onClick={onClickViewOnBlockExplorer('contract', selectedAsset.contractAddress, selectedAsset.tokenId)}>
+                    <ExplorerIcon />
+                  </ExplorerButton>
+                </ProjectDetailIDRow>
+              </DetailSectionColumn>
+            </DetailSectionRow>
+            <ProjectDetailRow>
+              {/* TODO: Add nft logo when data is available from backend: https://github.com/brave/brave-browser/issues/22627 */}
+              {/* <ProjectDetailImage src={nftMetadata.contractInformation.logo} /> */}
+              <ProjectDetailName>{nftMetadata.contractInformation.name}</ProjectDetailName>
+              {nftMetadata.contractInformation.website && nftMetadata.contractInformation.twitter && nftMetadata.contractInformation.facebook &&
+                <ProjectDetailButtonRow>
+                  <ProjectDetailButton onClick={onClickWebsite}>
+                    <ProjectWebsiteIcon/>
+                  </ProjectDetailButton>
+                  <ProjectDetailButtonSeperator/>
+                  <ProjectDetailButton onClick={onClickTwitter}>
```

Comment:
Probable XSS here with the onClickHandler

Reply:
looks fixed